rex/multi_year_resource.py

```diff
@@ -279,14 +318,18 @@ def close(self):
         """
         Close all h5py.File instances
         """
-        for f in self._h5_map.values():
+        for f in self._h5_map['h5']:
             f.close()
 
 
 class MultiYearResource(MultiTimeResource):
     """
     Class to handle multiple years of resource data stored accross multiple
-    .h5 files
+    .h5 files. This also works if each year is split into multiple files each
+    containing different datasets (e.g. for Sup3rCC and hi-res WTK+NSRDB).
```

Comment:
Sorry for being dense here. This only allows a single year to be split across multiple files if the datasets are different, right? We still don't allow to split within a year?

So for example, we *do* allow this configuration:
 - file 1: 2012 data for windspeed 
 - file 2; 2012 data for wind direction

and we *don't* allow this configuration:
 - file 1: Jan - June 2012 data for wind speed and wind direction
 - file 2: July - Dec 2012 data for wind speed and wind direction

Is that right? In either case, we should consider making this docstring even more clear with this regard.

Reply:
I'm not sure why it was originally designed this way but the multi-time handlers do multiple files within a year. Will add to the docstring.